libr/include/r_util/r_str.h

```diff
@@ -19,6 +19,51 @@ typedef enum {
 	R_STRING_ENC_GUESS = 'g',
 } RStrEnc;
 
+// r_strf API can be used for short strings that are needed for a short period
+// of time. They are allocated on the stack as needed, by making sure the total
```

Comment:
tldr comment. remove it

Reply:
why? it explains what is the purpose of these functions and when they should be used. I think it is *very* useful.

Reply:
tldr you can probably explain this in 2 or 3 lines and we should put this documentation into the book in the developer section. or maybe in the DEVELOPERS:md instead of messing the code with non-compilable chars